exampm fails to build in spack

`spack install exampm` leads to:
```
[ 66%] Building CXX object examples/CMakeFiles/DamBreak.dir/dam_break.cpp.o
cd /tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-build/examples && /home/junghans/spack.git/lib/spack/env/gcc/g++   -I/tmp/junghans/spack-stage/spack-stage-exa
mpm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-build/examples -I/tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/examples -I/tmp/junghans/spack-stage/spack
-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/src -I/tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-build/src -isystem /home/junghans/spack.git
/opt/spack/linux-centos7-haswell/gcc-8.2.0/kokkos-3.1.01-ewfgo3wepuf7d37jgtinll7qk6zigof7/include -isystem /home/junghans/spack.git/opt/spack/linux-centos7-haswell/gcc-8.2.0/openmpi-3.1.6-22nn2lya7jfgf7eo
4vw42n7aq6t7dz2q/include -isystem /home/junghans/spack.git/opt/spack/linux-centos7-haswell/gcc-8.2.0/cabana-master-ueinsnbtht7zrp5rmwof3wrt4jkd3pq7/include -isystem /home/junghans/spack.git/opt/spack/linu
x-centos7-haswell/gcc-8.2.0/silo-4.10.2-bk5whiftxlqnmvjcigaz5466nmm6vhaf/include  -O2 -g -DNDEBUG   -march=core-avx2 -mtune=core-avx2 -pthread -o CMakeFiles/FreeFall.dir/free_fall.cpp.o -c /tmp/junghans/s
pack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/examples/free_fall.cpp
cd /tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-build/examples && /home/junghans/spack.git/lib/spack/env/gcc/g++   -I/tmp/junghans/spack-stage/spack-stage-exa
mpm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-build/examples -I/tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/examples -I/tmp/junghans/spack-stage/spack
-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/src -I/tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-build/src -isystem /home/junghans/spack.git
/opt/spack/linux-centos7-haswell/gcc-8.2.0/kokkos-3.1.01-ewfgo3wepuf7d37jgtinll7qk6zigof7/include -isystem /home/junghans/spack.git/opt/spack/linux-centos7-haswell/gcc-8.2.0/openmpi-3.1.6-22nn2lya7jfgf7eo
4vw42n7aq6t7dz2q/include -isystem /home/junghans/spack.git/opt/spack/linux-centos7-haswell/gcc-8.2.0/cabana-master-ueinsnbtht7zrp5rmwof3wrt4jkd3pq7/include -isystem /home/junghans/spack.git/opt/spack/linu
x-centos7-haswell/gcc-8.2.0/silo-4.10.2-bk5whiftxlqnmvjcigaz5466nmm6vhaf/include  -O2 -g -DNDEBUG   -march=core-avx2 -mtune=core-avx2 -pthread -o CMakeFiles/DamBreak.dir/dam_break.cpp.o -c /tmp/junghans/s
pack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/examples/dam_break.cpp
In file included from /tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/src/ExaMPM_Solver.hpp:16,
                 from /tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/examples/free_fall.cpp:1:
/tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/src/ExaMPM_TimeIntegrator.hpp: In instantiation of 'void ExaMPM::TimeIntegrator::fieldSolve(const ExecutionSp
ace&, const ProblemManagerType&, double, double, const ExaMPM::BoundaryCondition&) [with ProblemManagerType = ExaMPM::ProblemManager<Kokkos::HostSpace>; ExecutionSpace = Kokkos::Serial]':
/tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/src/ExaMPM_TimeIntegrator.hpp:373:15:   required from 'void ExaMPM::TimeIntegrator::step(const ExecutionSpace
&, const ProblemManagerType&, double, double, const ExaMPM::BoundaryCondition&) [with ProblemManagerType = ExaMPM::ProblemManager<Kokkos::HostSpace>; ExecutionSpace = Kokkos::Serial]'
/tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/src/ExaMPM_Solver.hpp:97:33:   required from 'void ExaMPM::Solver<MemorySpace, ExecutionSpace>::solve(double,
 int) [with MemorySpace = Kokkos::HostSpace; ExecutionSpace = Kokkos::Serial]'
/tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/src/ExaMPM_Solver.hpp:79:10:   required from here
/tmp/junghans/spack-stage/spack-stage-exampm-master-rqaueribiehmdob6zd7sxbj56rls54wu/spack-src/src/ExaMPM_TimeIntegrator.hpp:134:10: error: no matching function for call to 'Cajita::LocalGrid<Cajita::UniformMesh<double> >::indexSpace(Cajita::Ghost, Cajita::Node, Cajita::Global)'
     auto global_nodes =
          ^~~~~~~~~~~~
```

--- a/src/ExaMPM_TimeIntegrator.hpp
+++ b/src/ExaMPM_TimeIntegrator.hpp
@@ -131,15 +131,16 @@ void fieldSolve( const ExecutionSpace& exec_space,
     double mass_epsilon = 1.0e-12;
 
     // Compute the velocity.
-    auto global_nodes =
+    auto l2g = Cajita::IndexConversion::createL2G(
+        *(pm.mesh()->localGrid()), Cajita::Node() );
+    auto local_nodes =
         pm.mesh()->localGrid()->indexSpace(
-            Cajita::Ghost(), Cajita::Node(), Cajita::Global() );
+            Cajita::Ghost(), Cajita::Node(), Cajita::Local() );
     Kokkos::parallel_for(
-        Cajita::createExecutionPolicy(global_nodes,exec_space),
-        KOKKOS_LAMBDA( const int gi, const int gj, const int gk ){
-            auto li = gi - global_nodes.min(0);
-            auto lj = gj - global_nodes.min(1);
-            auto lk = gk - global_nodes.min(2);
+        Cajita::createExecutionPolicy(local_nodes,exec_space),
+        KOKKOS_LAMBDA( const int li, const int lj, const int lk ){
+            int gi, gj, gk;
+            l2g( li, lj, lk, gi, gj, gk );
 
             // Only compute velocity if a node has mass
             u_i(li,lj,lk,0) =
@@ -327,15 +328,16 @@ void correctParticlePositions( const ExecutionSpace& exec_space,
 
     // Apply boundary condition to position correction.
     // Compute the velocity.
-    auto global_nodes =
+    auto l2g = Cajita::IndexConversion::createL2G(
+        *(pm.mesh()->localGrid()), Cajita::Node() );
+    auto local_nodes =
         pm.mesh()->localGrid()->indexSpace(
-            Cajita::Ghost(), Cajita::Node(), Cajita::Global() );
+            Cajita::Ghost(), Cajita::Node(), Cajita::Local() );
     Kokkos::parallel_for(
-        Cajita::createExecutionPolicy(global_nodes,exec_space),
-        KOKKOS_LAMBDA( const int gi, const int gj, const int gk ){
-            auto li = gi - global_nodes.min(0);
-            auto lj = gj - global_nodes.min(1);
-            auto lk = gk - global_nodes.min(2);
+        Cajita::createExecutionPolicy(local_nodes,exec_space),
+        KOKKOS_LAMBDA( const int li, const int lj, const int lk ){
+            int gi, gj, gk;
+            l2g( li, lj, lk, gi, gj, gk );
             bc( gi, gj, gk,
                 x_i(li,lj,lk,0), x_i(li,lj,lk,1), x_i(li,lj,lk,2) );
         });

